api/src/opentrons/hardware_control/instruments/ot3/pipette_handler.py

```diff
@@ -723,23 +711,41 @@ def build_one_shake() -> List[Tuple[top_types.Point, Optional[float]]]:
 
         return []
 
-    def plan_check_pick_up_tip(
+    def plan_ht_pick_up_tip(self, mount) -> TipActionSpec:
```

Comment:
```suggestion
    def plan_ht_pick_up_tip(self, mount: OT3Mount) -> TipActionSpec:
```
? 
heck it might not even need the mount